<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdressBookEntry</class>
 <widget class="QWidget" name="AdressBookEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdressBookEntry</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_SecondName">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_SecondName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_FirstName">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_FirstName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_thirdName">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_thirdName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_address">
       <property name="text">
        <string>Адрес</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_address"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_add_te">
       <property name="text">
        <string>Доп тел.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_add_tel"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_mobile_tel">
       <property name="text">
        <string>Моб тел.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_mobile_tel"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_other">
       <property name="text">
        <string>Другое</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QPlainTextEdit" name="plainTextEdit_other"/>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton_save">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
